### PR TITLE
Hide cancel button on finished pipelinerun

### LINKED ISSUE #444: Cancel button on finished pipeline
I have used tutorial from https://github.com/openshift/pipelines-tutorial#deploy-sample-application.

When the pipeline finished, there was still cancel button on this pipeline:
image

I suppose the cancel button should disappear, when the pipeline finished.

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #444

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -666,12 +666,12 @@
       "view/item/context": [
         {
           "command": "k8s.tekton.run.logs",
-          "when": "viewItem =~ /vsKubernetes\\.resource\\.(pipelinerun|taskrun)/i",
+          "when": "viewItem =~ /vsKubernetes\\.resource\\.(pipelinerun|pr|taskrun)/i",
```
Comment (evidolob) [inline package.json:669]: I think we need something more descriptible that just `pr`

File: package.json [hunk 2]
```diff
@@ -666,12 +666,12 @@
       "view/item/context": [
         {
           "command": "k8s.tekton.run.logs",
-          "when": "viewItem =~ /vsKubernetes\\.resource\\.(pipelinerun|taskrun)/i",
+          "when": "viewItem =~ /vsKubernetes\\.resource\\.(pipelinerun|pipelineRun|taskrun)/i",
```
Comment (evidolob) [inline package.json:669]: It looks like mistake `pipelinerun`  and `pipelineRun`, what the difference between them?

Comment (sudhirverma) [reply package.json:669]: To hide the cancel button I am using `pipelineRun`, I will switch the contextValue to `pipelineRun` when pipelinerun is finished.

Comment (evidolob) [reply package.json:669]: So, maybe `pipelineRunFinished` is more descriptive?

Comment (sudhirverma) [reply package.json:669]: we are using `pipelineRun` to one file in the editor. So I don't think `pipelineRunFinished` would be good to use here.

Comment (evidolob) [reply package.json:669]: OK, anyway having `pipelinerun` and `pipelineRun` is not good, you need to rename one of it.

**Review by evidolob — APPROVED** [review]

**Pull request merged**